Effisense/sensor-dashboard
# Feat/#57 fix design container sensor view

Closes: #57 

## Checklist


## Changelog
- Update design on view for container with Figma prototype
- Update design on view for sensor with Figma prototype

## How to test
- Go to the view for one specific sensor and check that the design is looking good, check that the page is responsive aswell
- Go to the view for one specific container and check that the design is looking good, check that the page is responsive aswell
